addons/mail/static/tests/composer/composer_tests.js

```diff
@@ -436,6 +436,41 @@ QUnit.test("pending mentions are kept when toggling composer", async (assert) =>
     assert.containsOnce($, ".o-mail-Message-body a.o_mail_redirect:contains(@Mitchell Admin)");
 });
 
+QUnit.test('suggestion list is switchable between commands', async (assert) => {
```

Comment:
The name of test is poor: we can't grasp what's the general idea of the test without looking at its content.

After looking at the test, it looks like you want suggestion prompt to match user selection in input.
Test could be named _"Composer suggestion should match with input selection"_

Reply:
Well, actually, the bug is just `suggestion list is not able to switch if there is already an instance of it` but ok, I think your name also working correctly!